imagetest/cmd/manager/main.go

```diff
@@ -38,6 +39,7 @@ var (
 	parallelCount = flag.Int("parallel_count", 5, "TestParallelCount")
 	filter        = flag.String("filter", "", "only run tests matching filter")
 	exclude       = flag.String("exclude", "", "skip tests matching filter")
+	extraTests    = flag.String("extra_tests", "", "run extra tests which would normally not be run and do not block releases")
```

Comment:
I'm wondering if this flag makes sense since we already have filter (which is basically include only) and exclude. The choice of which test suites to run is left to the place it is being instrumented (in our CI pipeline or elsewhere for example).

Reply:
storage_perf is now excluded in the concourse files